src/pages/workspaces/workspace/Tools.js

```diff
@@ -269,15 +290,24 @@ export const Tools = _.flow(
     return _.find({ namespace, name }, configs)
   }
 
+  addRedactedAttribute(config, methods) {
+    const { methodName, methodNamespace, methodVersion, sourceRepo } = config.methodRepoMethod
+    const isRedacted = _.some({ name: methodName, namespace: methodNamespace, snapshotId: methodVersion }, methods)
```

Comment:
nit: If we inline this in the conditional below, we could avoid doing this work for non-agora methods.